src/CodeStyle/Core/Analyzers/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,74 @@
-
+Microsoft.CodeAnalysis.CodeStyle.CodeStyleOption<T>
```

Comment:
Do we really want to add these public APIs to this layer? If so, `Microsoft.CodeAnalysis.Options.IOption` does not seem to belong here, as it is not related to code style. For example, if we were to tackle [https://github.com/dotnet/roslyn/issues/31225](https://github.com/dotnet/roslyn/issues/31225), these probably go into a non-code style layer?

Reply:
Nothing references this layer right now. Once .editorconfig is provided by the compiler, we'll move the formatter implementation to use `AnalyzerConfigOptions` instead of `OptionSet` and eliminate the need to duplicate the `IOption` types in this layer.

Reply:
Sounds good, thanks!

---
In reply to: [236402494](https://github.com/dotnet/roslyn/pull/31276#discussion_r236402494) [](ancestors = 236402494)